ConsoleApp/Point.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+
+namespace Snake
+{
+    class Point
+    {
+        public int x;
+        public int y;
+        public char sym;
+        public Point()
+        {
+        }
+        public Point(int _x, int _y, char _sym)
```

Comment:
code convention violated here
```suggestion
        public Point(int x, int y, char sum)
```